### PR TITLE
Tests fix after examples refactor

### PR DESCRIPTION
**Pull request by szalpal**
Signed-off-by: Michał Szołucha <mszolucha@nvidia.com>

#### Why we need this PR?
Cause after examples refactor nightlies failed

#### What happened in this PR?
Fixed paths for notebooks and examples

### CODE REVIEW

**Comment by szalpal** [comment]
!build

File: qa/TL3_RN50_convergence/test_paddle.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ function CLEAN_AND_EXIT {
 export USE_CUDA_VERSION=$(cat /usr/local/cuda/version.txt | head -1 | sed 's/.*Version \([0-9]\+\)\.\([0-9]\+\).*/\1\2/')
 pip install $(python /opt/dali/qa/setup_packages.py -i 0 -u paddle --cuda ${USE_CUDA_VERSION})
 
-cd /opt/dali/docs/examples/paddle/resnet50
+cd /opt/dali/docs/examples/frameworks/paddle/resnet50
```
Comment (JanuszL) [inline qa/TL3_RN50_convergence/test_paddle.sh]: Why not to move paddle into use_cases?

File: qa/TL1_paddle_tsm/test.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 # used pip packages
 pip_packages="numpy paddle"
-target_dir=./docs/examples/paddle/tsm/
+target_dir=./docs/examples/frameworks/paddle/tsm/
```
Comment (JanuszL) [inline qa/TL1_paddle_tsm/test.sh]: Why not move it to use_cases?

File: qa/TL1_paddle_ssd/test.sh [hunk 3]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash -e
 # used pip packages
 pip_packages="numpy paddle"
-target_dir=./docs/examples/paddle/ssd/
+target_dir=./docs/examples/frameworks/paddle/ssd/
```
Comment (JanuszL) [inline qa/TL1_paddle_ssd/test.sh]: Why not to move use_cases?

File: qa/TL1_jupyter_plugins/test_tf.sh [hunk 4]
```diff
@@ -15,7 +15,7 @@ test_body() {
 
 
   # test code
-  find tensorflow/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "tensorflow-*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_tf.sh]: I wouldn't relay on all tf jupyters starting from `tensorflow-`.

File: qa/TL1_jupyter_plugins/test_pytorch.sh [hunk 5]
```diff
@@ -14,11 +14,11 @@ test_body() {
   black_list_files="#"
 
   # test code
-  find pytorch/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "pytorch-*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_pytorch.sh]: Same as TF

File: qa/TL1_jupyter_plugins/test_paddle.sh [hunk 6]
```diff
@@ -13,7 +13,7 @@ test_body() {
   black_list_files="#"
 
   # test code
-  find paddle/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "paddle-*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_paddle.sh]: Same as for TF

File: qa/TL1_jupyter_plugins/test_mxnet.sh [hunk 7]
```diff
@@ -13,7 +13,7 @@ test_body() {
   black_list_files="#"
 
   # test code
-  find mxnet/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "mxnet-*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_mxnet.sh]: Same as for TF

File: qa/TL1_paddle_ssd/test.sh (hunk 3 above)
Comment (szalpal) [reply qa/TL1_paddle_ssd/test.sh]: Done

File: qa/TL1_paddle_tsm/test.sh (hunk 2 above)
Comment (szalpal) [reply qa/TL1_paddle_tsm/test.sh]: Done

File: qa/TL3_RN50_convergence/test_paddle.sh (hunk 1 above)
Comment (szalpal) [reply qa/TL3_RN50_convergence/test_paddle.sh]: Done

File: qa/TL1_jupyter_plugins/test_paddle.sh [hunk 8]
```diff
@@ -2,7 +2,7 @@
 
 # used pip packages
 pip_packages="jupyter matplotlib paddle"
-target_dir=./docs/examples
+target_dir=./docs/examples/frameworks/paddle
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_paddle.sh]:
```suggestion
target_dir=./docs/examples/
```

File: qa/TL1_jupyter_plugins/test_paddle.sh [hunk 9]
```diff
@@ -13,7 +13,7 @@ test_body() {
   black_list_files="#"
 
   # test code
-  find paddle/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_paddle.sh]:
```suggestion
  find frameworks/mxnet -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```

File: qa/TL1_jupyter_plugins/test_tf.sh [hunk 10]
```diff
@@ -2,7 +2,7 @@
 
 # used pip packages
 pip_packages="jupyter matplotlib tensorflow-gpu"
-target_dir=./docs/examples
+target_dir=./docs/examples/frameworks/tensorflow
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_tf.sh]:
```suggestion
target_dir=./docs/examples/
```

File: qa/TL1_jupyter_plugins/test_tf.sh [hunk 11]
```diff
@@ -15,7 +15,7 @@ test_body() {
 
 
   # test code
-  find tensorflow/* -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
+  find . -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```
Comment (JanuszL) [inline qa/TL1_jupyter_plugins/test_tf.sh]:
```suggestion
  find /frameworks/tensorflow -name "*.ipynb" | sed "/${black_list_files}/d" | xargs -i jupyter nbconvert \
```

**Comment by szalpal** [comment]
!build

**Comment by JanuszL** [comment]
!build

**Comment by klecki** [comment]
There is L1_TL1_tensorflow-dali_test job that keeps failing with this change, but it's allowed to fail in CI, so the job was marked as passed.

**Comment by JanuszL** [comment]
> There is L1_TL1_tensorflow-dali_test job that keeps failing with this change, but it's allowed to fail in CI, so the job was marked as passed.

I hope it is fixed now.

**Pull request merged**